apps/docs/content/getting-started/animation.mdx

```diff
@@ -0,0 +1,247 @@
+---
+title: Animation
+storybook: '/brand/storybook/?path=/story/components-animations--playground'
+description: Use animations to add visual interest and interactivity to a web page or application.
+---
+
+import {PropTableValues} from '../../src/components'
+import {Link} from 'gatsby'
+import {AnimationVariants, Label} from '@primer/react-brand'
+import {supportedComponents} from '../../scripts/components-with-animation'
+
```

Comment:
We could add a title as well as we do for the Animation component section

```suggestion
### Animation props
```

Reply:
Added a header, but slightly tweaked it for accuracy. Thanks Josep.